.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3
```

Comment:
Only clarification question: we don't have NOTICE_3RD_CONTENT in packages... Is it intentional?

Reply:
no, we still have this as open task. It is not that easy to fully integrate the license checker because of some transitive dependencies

Reply:
I already started this some time ago (check branch 'fix/licenses')

Reply:
If we have it in backlog - fine for me